core/src/channel_group/mod.rs

```diff
@@ -59,19 +59,22 @@ impl ChannelGroup {
             ),
         };
 
-        for i in 0..config.channel_count {
-            let mut init = config.channel_init_options;
-            init.drums_only = config.drums_channels.clone().into_iter().any(|c| c == i);
-
+        for _ in 0..config.channel_count {
             channels.push(VoiceChannel::new(
-                init,
+                config.channel_init_options,
                 config.audio_params,
                 channel_pool.clone(),
             ));
             channel_events_cache.push(Vec::new());
             sample_cache_vecs.push(Vec::new());
         }
 
+        if config.channel_count >= 16 {
```

Comment:
Just for later, but should we make this optional in the future? Just a config flag or something if it's not too messy

Reply:
I mean we could, although the user always will have the option to send a config event to channel 10 to make it standard if they wish.

One idea would also be to manage the channel count with a type or something
```rust
enum SynthFormat {
    // Will create 16 channels, with 10 being drums
    Midi,

    // Probably other formats here? Idk what else there is

    // Will create the amount of channels without setting any to drums
    // so the user can do it themselves
    Custom { channels: u32 }.
}
```

Reply:
Yeah that works